sdks/wasm/packager.cs

```diff
@@ -240,8 +251,8 @@ enum CopyType
 
 		//are we working from the tree?
 		if (sdkdir != null) {
-			framework_prefix = tool_prefix; //all framework assemblies are currently side built to packager.exe
-			bcl_prefix = Path.Combine (sdkdir, "wasm-bcl/wasm");
+			framework_prefix = Path.Combine (tool_prefix, "framework"); //all framework assemblies are currently side built to packager.exe
```

Comment:
The --mono-sdkdir argument is supposed to point to sdks/out in the mono build tree.

Reply:
Got it, they what parameter should be used when using the packager from the build artifacts ?

Reply:
I think either the archive structure should be changed to mirror the tree structure (i.e. the bcl in wasm-bcl/wasm), or no argument should be required, i.e. the
		} else if (Directory.Exists (Path.Combine (tool_prefix, "../out/wasm-bcl/wasm"))) {
			framework_prefix = tool_prefix; //all framework assemblies are currently side built to packager.exe
block should be modified to work with the build artifacts.

Reply:
The archive structure was modified for issue https://github.com/mono/mono/issues/11344 and PR: https://github.com/mono/mono/pull/11347

Reply:
Indeed, I've made the updates to remove those changes.